README.md

```diff
@@ -1,2 +1,20 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+a - Checks if a list has 2 or more of the same items inside it.
+
+b - Because it would take a lot longer to calculate.
+
+c - Because j only ever needs to check the numbers i has been through.
+
+d - Because j doesn't need to go through the whole list every time.
```

Comment:
✔️ but why specifically does the algorithm speed up by approximately a factor of 2?